src/endpoints/cli.js

```diff
@@ -41,6 +41,23 @@ const download = async (req, res) => {
 };
 
 
+const installer = (req, res) => {
+  const os = req.params.os;
+  switch (os) {
+    case "linux":
+    case "mac":
+      return res.redirect("https://github.com/nextstrain/cli/raw/master/bin/standalone-installer-unix");
+
+    case "windows":
+      return res.redirect("https://github.com/nextstrain/cli/raw/master/bin/standalone-installer-windows");
```

Comment:
_(non-blocking)_

Future-proof in case the default branch ever changes:

```suggestion
    case "linux":
    case "mac":
      return res.redirect("https://github.com/nextstrain/cli/raw/HEAD/bin/standalone-installer-unix");

    case "windows":
      return res.redirect("https://github.com/nextstrain/cli/raw/HEAD/bin/standalone-installer-windows");
```

Reply:
Ah, good call. As a cute trick it seems even `@` is supported, but I'll spell it out `HEAD`. :-)

Reply:
Updated with repush.

Reply:
Also, `-` sometimes works too (e.g. https://github.com/nextstrain/nextstrain.org/blob/-/README.md) and doesn't redirect to an exact commit.

Source: https://stackoverflow.com/a/65158264

Reply:
Makes sense to me, as `-` can be used with `git` commands to mean `the last ref I had checked out`, e.g. `git checkout -`, which is similar in spirit to `cd -`.